code/modules/mechs/components/head.dm

```diff
@@ -88,6 +88,9 @@
 	desc = "A primitive set of sensors designed to provide basic visual information to the pilot."
 	max_damage = 125
 	power_use = 0
+	armor = list(melee = 20, bullet = 10, energy = 5, bomb = 60, bio = 100, rad = 0)
+	shielding = 5
+	front_mult = 0.9
```

Comment:
If sensors take more damage from the front, make sure to mention that not just in the component, but all mech descriptions in general, for the laity.